delegation-toolkit/index.md

```diff
@@ -1,37 +1,29 @@
 ---
 title: Delegation Toolkit introduction
 sidebar_label: Introduction
-description: High-level overview of the Delegation Toolkit, its benefits, and where to start in the documentation.
+description: High-level overview of MetaMask Smart Accounts and the Delegation Toolkit.
 sidebar_position: 1
 ---
 
 import CardList from "@site/src/components/CardList"
 
-# Embed smart accounts using the Delegation Toolkit
+# Embed MetaMask Smart Accounts using the Delegation Toolkit
 
-The MetaMask Delegation Toolkit is a [Viem](https://viem.sh/)-based collection of tools for integrating
-embedded smart contract wallets, known as [MetaMask smart accounts](concepts/smart-accounts.md),
-into dapps. Developers can create and manage MetaMask smart accounts that delegate specific
-permissions, such as spending limits or time-based access, to other accounts.
+The MetaMask Delegation Toolkit is a [Viem](https://viem.sh/)-based collection of tools for embedding [MetaMask Smart Accounts](concepts/smart-accounts.md) into dapps.
+Smart accounts support programmable account behavior and advanced features like delegated permissions, multi-signature approvals, and gas abstraction.
 
-At the core of the toolkit is the [Delegation Framework](concepts/delegation.md#delegation-framework), a
-set of open-source, audited smart contracts that manage the delegation lifecycle.
-
-Permissions are enforced through [caveats](concepts/caveat-enforcers.md), which are rule-based
-constraints that define the conditions of a delegation. The toolkit includes
-[built-in caveat enforcers](reference/caveats.md) for common
-use cases. It also supports [custom caveat enforcers](how-to/create-delegation/create-custom-caveat-enforcer.md)
-for advanced scenarios.
+The toolkit's [Delegation Framework](concepts/delegation.md#delegation-framework) extends MetaMask Smart Accounts with secure, rule-based permission sharing.
```

Comment:
I feel it can be more clear. The below also clarifies Delegations are core feature of MetaMask Smart Accounts, which we want to showcase in our docs, WDYT?

```
Delegations are a core feature of MetaMask Smart Accounts, enabling secure, rule-based permission sharing. These delegations are powered by the toolkit's Delegation Framework which defines how permissions are created, shared, and enforced.
```